static/app/components/onboardingWizard/filterSupportedTasks.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  feedbackOnboardingPlatforms,
+  replayOnboardingPlatforms,
+  withoutPerformanceSupport,
+} from 'sentry/data/platformCategories';
+import {type OnboardingTask, OnboardingTaskKey, type Project} from 'sentry/types';
+
+const replayRelatedTasks = [OnboardingTaskKey.SESSION_REPLAY];
+const performanceRelatedTasks = [
+  OnboardingTaskKey.FIRST_TRANSACTION,
+  OnboardingTaskKey.PERFORMANCE_GUIDE,
+  OnboardingTaskKey.METRIC_ALERT,
+];
+const feedbackRelatedTasks = [OnboardingTaskKey.USER_REPORTS];
```

Comment:
Are constants supposed to be capitalized? I'm not sure what we use in our TS/JS world haha

Reply:
I never thought about it. It is kind of confusing. `const something = somethingElse` should not be capitalized but hardcoded ones `const int MINS_IN_HR = 60` should be.